pageserver/src/consumption_metrics.rs

```diff
@@ -307,7 +313,7 @@ async fn calculate_synthetic_size_worker(
 
         let res = tokio::time::timeout_at(
             started_at + synthetic_size_calculation_interval,
-            task_mgr::shutdown_token().cancelled(),
+            cancel.cancelled(),
```

Comment:
Not a regression, the `task_mgr::shutdown_token()` is dragged from the `pageserver/src/bin/pageserver.rs` caller.